simapp/app.go

```diff
@@ -351,8 +352,18 @@ func NewSimApp(
 	app.mm.SetOrderBeginBlockers(
 		upgradetypes.ModuleName, capabilitytypes.ModuleName, minttypes.ModuleName, distrtypes.ModuleName, slashingtypes.ModuleName,
 		evidencetypes.ModuleName, stakingtypes.ModuleName,
+		authtypes.ModuleName, banktypes.ModuleName, govtypes.ModuleName, crisistypes.ModuleName, genutiltypes.ModuleName,
+		authz.ModuleName, feegrant.ModuleName, nft.ModuleName,
+		paramstypes.ModuleName, vestingtypes.ModuleName,
```

Comment:
this adds all modules even those that don't have any begin blockers?

Reply:
yeah. At least you're sure you won't forget

Reply:
I guess this is fine, makes the ux of these a bit weird since it may confuse people that bank is before elsewhere but the last here. But since this is being refactored its an okay stop gap.